.ci/oci-devworkspace-happy-path.sh

```diff
@@ -0,0 +1,170 @@
+#!/bin/bash
+#
+# Copyright (c) 2012-2021 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+
+#!/usr/bin/env bash
+# exit immediately when a command fails
+set -e
+# only exit with zero if all commands of the pipeline exit successfully
+set -o pipefail
+# error on unset variables
+set -u
+# print each command before executing it
+set -x
+
+trap 'Catch_Finish $?' EXIT SIGINT
+
+# Catch the finish of the job and write logs in artifacts.
+function Catch_Finish() {
+    # grab devworkspace-controller namespace events after running e2e
+    getDevWorkspaceOperatorLogs
+}
+
+# ENV used by PROW ci
+export CI="openshift"
+export ARTIFACTS_DIR=${ARTIFACT_DIR:-"/tmp/artifacts-che"}
+export NAMESPACE="eclipse-che"
+export DEVWORKSPACE_CONTROLLER_NAMESPACE="devworkspace-controller"
+export OPERATOR_REPO=$(dirname $(dirname $(readlink -f "$0")));
+export HAPPY_PATH_POD_NAME=happy-path-che
+export HAPPY_PATH_DEVFILE='https://gist.githubusercontent.com/l0rd/71a04dd0d8c8e921b16ba2690f7d5a47/raw/d520086e148c359b18c229328824dfefcf85e5ef/spring-petclinic-devfile-v2.0.0.yaml'
+
+# Pod created by openshift ci don't have user. Using this envs should avoid errors with git user.
+export GIT_COMMITTER_NAME="CI BOT"
+export GIT_COMMITTER_EMAIL="ci_bot@notused.com"
+
+function getDevWorkspaceOperatorLogs() {
+    mkdir -p ${ARTIFACTS_DIR}/devworkspace-operator
+    cd ${ARTIFACTS_DIR}/devworkspace-operator
+    for POD in $(oc get pods -o name -n ${DEVWORKSPACE_CONTROLLER_NAMESPACE}); do
+       for CONTAINER in $(oc get -n ${DEVWORKSPACE_CONTROLLER_NAMESPACE} ${POD} -o jsonpath="{.spec.containers[*].name}"); do
+            echo ""
+            echo "<=========================Getting logs from container $CONTAINER in $POD==================================>"
+            echo ""
+            oc logs ${POD} -c ${CONTAINER} -n ${DEVWORKSPACE_CONTROLLER_NAMESPACE} | tee $(echo ${POD}-${CONTAINER}.log | sed 's|pod/||g')
+        done
+    done
+    echo "======== oc get events ========"
+    oc get events -n ${DEVWORKSPACE_CONTROLLER_NAMESPACE}| tee get_events.log
+
+  mkdir -p ${ARTIFACTS_DIR}
+  /tmp/chectl/bin/chectl server:logs --chenamespace=${NAMESPACE} --directory=${ARTIFACTS_DIR} --telemetry=off
+}
+
+installChectl() {
+  curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl && \
+  chmod +x ./kubectl 
```

Comment:
why do we need kubectl?

Reply:
You are right. `oc` client is enough here.